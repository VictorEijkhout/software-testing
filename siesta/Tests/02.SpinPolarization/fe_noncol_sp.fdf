%include ../base_fe3.fdf
SystemLabel         fe_noncol_sp

%block kgrid_Monkhorst_Pack
   1  0  0  0.0
   0  1  0  0.0
   0  0  2  0.0
%endblock kgrid_Monkhorst_Pack

Spin non-collinear
Spin.Spiral.Scale ReciprocalLatticeVectors
%block Spin.Spiral
   0.0 0.1 0.1
%endblock
