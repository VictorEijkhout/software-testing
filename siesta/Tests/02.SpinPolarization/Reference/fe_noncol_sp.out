Executable      : /home/fnpedron/Siesta/siesta-dev/_build_serial/Src/siesta
Version         : 5.2.0-15-g8c00451d2
Architecture    : x86_64
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch -O3 -march=native
Environment variables: 
  SIESTA_PS_PATH: /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos
Parallelisations: none
NetCDF support
NetCDF-4 support
Lua support
DFT-D3 support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/_build_serial/Tests/02.SpinPolarization/fe_noncol_sp/work
* Running in serial mode.
>> Start of run:  17-JAN-2025  12:35:05

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

 NOTE: Siesta V5 introduces some new defaults for basis-set generation
 NOTE: which might lead to slightly different numerical results.
 NOTE: Please see the manual for full details and compatibility measures.


reinit: Reading from file ../../fe_noncol_sp.fdf

reinit: -----------------------------------------------------------------------
reinit: System Name: Linear Fe3 cluster
reinit: -----------------------------------------------------------------------
reinit: System Label: fe_noncol_sp
reinit: -----------------------------------------------------------------------

NOTE on units for spin output: 
{S} is the spin magnetic moment, in units of the Bohr magneton,
assuming a g-factor for the electron spin of exactly 2.
The spin magnetic moment is then numerically equal to the charge inbalance,
in units of electrons, between the spin-up and -down channels.


initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   26 Label: Fe_nc

Ground state valence configuration (from tables):   4s02  4p00  3d06  4f00

---- Processing specs for species: Fe_nc
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Fe_nc.psf

---- Pseudopotential check for Fe

Pseudized shells:
4s( 1.00) rc: 2.00
4p( 0.00) rc: 2.00
3d( 7.00) rc: 2.00
4f( 0.00) rc: 2.00
Valence configuration for ps generation: (assumed as above)
For Fe_nc, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Fe_nc                Z=  26    Mass=  55.845        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=4
          i=1  nzeta=1  polorb=1  (4s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=4
          i=1  nzeta=0  polorb=0  (4p)  (perturbative polarization orbital) (from 4s)
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3d)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Fe_nc                 (Z =  26)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    8.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  2.8472
V l=1 = -2*Zval/r beyond r=  2.8472
V l=2 = -2*Zval/r beyond r=  2.8472
V l=3 = -2*Zval/r beyond r=  2.8472
All V_l potentials equal beyond r=  1.9894
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.8472
Using large-core scheme (fit) for Vlocal

atom: Estimated core radius    2.84716
Fit of Vlocal with continuous 2nd derivative
Fitting vlocal at       1.9954
Choosing vlocal chloc cutoff:  5741  3.100644
qtot up to nchloc:    8.00000259
atom: Maximum radius for chlocal:    3.10064
atom: Maximum radius for r*vlocal+2*Zval:    3.10064

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.991366   el= -0.319855   Ekb=  3.826230   kbcos=  0.253712
GHOST: No ghost state for L =  1
   l= 1   rc=  2.019582   el= -0.070931   Ekb=  3.335986   kbcos=  0.155791
GHOST: No ghost state for L =  2
   l= 2   rc=  1.989365   el= -0.307264   Ekb=-12.632066   kbcos= -0.657060
GHOST: No ghost state for L =  3
   l= 3   rc=  2.015527   el=  0.019538   Ekb= -1.701666   kbcos= -0.006384

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    7.355669
                 energy =   -0.308198
                kinetic =    0.284874
    potential(screened) =   -0.593072
       potential(ionic) =   -5.763042

SPLIT: Orbitals with angular momentum L= 2

SPLIT: Basis orbitals for state 3d

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.584362
                 energy =   -0.296412
                kinetic =    8.773647
    potential(screened) =   -9.070058
       potential(ionic) =  -18.006145

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 4s
Split based on tail norm

   izeta = 1
                     rc =    7.355669
                 energy =    0.004656
                kinetic =    0.554633
    potential(screened) =   -0.549977
       potential(ionic) =   -5.337881
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 4s( 2.00)                                                            
 4p( 0.00)                                                            
 3d( 6.00)                                                            
Vna: chval, zval:    8.00000   8.00000

Vna:  Cut-off radius for the neutral-atom potential:   7.355669
comcore: Pseudo-core radius Rcore=  3.839063

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   26 Fe_nc                   # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Fe_nc                 2                    # Species label, number of l-shells
 n=4   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   7.356   
   1.000   
 n=3   2   1                         # n, l, Nzeta 
   4.584   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Fe_nc.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                            (in Bohr units)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   3.64400  1        1
siesta:      0.00000   0.00000   0.00000  1        2
siesta:      0.00000   0.00000  -3.64400  1        3

siesta: System type = bulk      
Using current reciprocal lattice vectors for Spin.Spiral.Scale
Beware any cell changes by the end of the run

initatomlists: Number of atoms, orbitals, and projectors:      3    27    48

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = non-collinear
redata: Number of spin components                   = 4
redata: Time-Reversal Symmetry                      = F
redata: Spin spiral                                 = T
redata: Spin spiral pitch wave vector               = 0.000000E+00 0.523599E-01 0.314159E-01
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken charges (when)               = geometry+end
redata: Write Mulliken Pop.                         = Atomic and Orbital charges
redata: Write Hirshfeld charges (when)              = none
redata: Write Voronoi charges (when)                = none
redata: Write spin charge (when)                    = scf
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   100.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =       20
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   300.0000 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 8
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
mix.SCF: Spin-component mixing                   all
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 8
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    24.000000
Total ionic charge:    24.000000
k-point displ. along   3 input, could be:     0.00    0.50

Time-reversal symmetry not used.

siesta: k-grid: Number of k-points = 2
siesta: k-points from Monkhorst-Pack grid
siesta: k-cutoff (effective) =     3.175 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     1     0     0      0.000
siesta: k-grid:     0     1     0      0.000
siesta: k-grid:     0     0     2      0.000

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     3 x     3 x     3  =      27
superc: Number of atoms, orbitals, and projectors:     81    729   1296


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     3 x     3 x     3  =      27
superc: Number of atoms, orbitals, and projectors:     81    729   1296

outcell: Unit cell vectors (Ang):
        6.350127    0.000000    0.000000
        0.000000    6.350127    0.000000
        0.000000    0.000000   10.583544

outcell: Cell vector modules (Ang)   :    6.350127    6.350127   10.583544
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    426.7720
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=27 nrows=27 sparsity=1.8340 nnzs=1337, refcount: 7>
  <dData1D:(new from dSpData1D) n=1337, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...
DM filled with atomic data (user-defined):
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=27 nrows=27 sparsity=1.8340 nnzs=1337, refcount: 8>
  <dData2D:DM n=1337 m=4, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      27

InitMesh: MESH = 40 x 40 x 64 = 102400
InitMesh: Mesh cutoff (required, used) =   100.000   101.065 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       232.505555
siesta: Eions   =      2102.576618
siesta: Ena     =        92.089899
siesta: Ekin    =      2810.190941
siesta: Enl     =     -1815.473089
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =      -116.936418
siesta: DUscf   =        55.128632
siesta: DUext   =         0.000000
siesta: Ex      =     -1117.149588
siesta: Ec      =       -97.949471
siesta: Exc     =     -1215.099058
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -2222.871165
siesta: Etot    =     -2292.675710
siesta: FreeEng =     -2292.765472

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1    -2222.871165    -2292.675710    -2292.765472  0.993965 -9.358835 34.426612
     spin moment: {S} , |S| = {    1.64541    0.00000   -0.00000 }     1.64541
timer: Routine,Calls,Time,% = IterSCF        1       0.075  17.85
   scf:    2    -2300.362555    -2296.730091    -2296.812843  0.097534 -9.211230 31.410702
     spin moment: {S} , |S| = {    1.51487    0.00006   -0.00000 }     1.51487
   scf:    3    -2281.688990    -2303.149285    -2303.217307  0.333926 -7.720021 20.962927
     spin moment: {S} , |S| = {    3.02654    0.00122    0.00000 }     3.02654
   scf:    4    -2145.143302    -2269.034951    -2269.053590  0.834306 -7.970006 45.118285
     spin moment: {S} , |S| = {    1.08695    0.00037   -0.00000 }     1.08695
   scf:    5    -2332.291489    -2309.740335    -2309.740335  0.559803 -3.538950 17.654864
     spin moment: {S} , |S| = {    0.88150    0.00132    0.00000 }     0.88150
   scf:    6    -2276.316219    -2319.445616    -2319.446656  0.370858 -3.741810 15.562649
     spin moment: {S} , |S| = {    1.70310    0.00140    0.00000 }     1.70310
   scf:    7    -2327.595879    -2323.862761    -2323.863510  0.044610 -3.947755 11.442461
     spin moment: {S} , |S| = {    1.68042    0.00137    0.00000 }     1.68042
   scf:    8    -2287.560824    -2318.745542    -2318.745542  0.274904 -2.115593 10.703863
     spin moment: {S} , |S| = {    1.05887    0.00150    0.00000 }     1.05887
   scf:    9    -2317.017787    -2317.896409    -2317.896409  0.007832 -2.230169 10.733785
     spin moment: {S} , |S| = {    1.03573    0.00150    0.00000 }     1.03573
   scf:   10    -2326.988125    -2322.749756    -2322.749757  0.046705 -3.008556  7.855734
     spin moment: {S} , |S| = {    1.12746    0.00168    0.00000 }     1.12746
   scf:   11    -2330.856101    -2328.565183    -2328.565262  0.104439 -4.012150  2.785368
     spin moment: {S} , |S| = {    1.42011    0.00169   -0.00000 }     1.42011
   scf:   12    -2298.197810    -2319.608651    -2319.609815  0.580402 -5.246540 12.208884
     spin moment: {S} , |S| = {    1.89701    0.00185    0.00000 }     1.89701
   scf:   13    -2317.609684    -2318.634842    -2318.635746  0.011356 -5.209544 12.552368
     spin moment: {S} , |S| = {    1.93561    0.00185    0.00000 }     1.93561
   scf:   14    -2320.600633    -2319.637678    -2319.639752  0.013126 -5.011836 11.883689
     spin moment: {S} , |S| = {    1.89688    0.00190    0.00000 }     1.89688
   scf:   15    -2315.432203    -2317.618565    -2317.619324  0.022729 -5.213345 12.908280
     spin moment: {S} , |S| = {    1.97030    0.00181    0.00000 }     1.97030
   scf:   16    -2328.214170    -2323.497056    -2323.547009  0.135186 -4.778283  8.917823
     spin moment: {S} , |S| = {    1.89952    0.00435    0.00000 }     1.89952
   scf:   17    -2329.611574    -2328.730311    -2328.745247  0.381889 -4.533816  2.410869
     spin moment: {S} , |S| = {    1.62676    0.00278   -0.00000 }     1.62676
   scf:   18    -2328.783608    -2328.797413    -2328.836717  0.044904 -4.610987  1.039741
     spin moment: {S} , |S| = {    1.68687    0.00400   -0.00000 }     1.68688
   scf:   19    -2328.889232    -2328.917908    -2328.926149  0.061057 -4.501355  0.760246
     spin moment: {S} , |S| = {    1.56457    0.00259    0.00000 }     1.56457
   scf:   20    -2321.897270    -2326.626274    -2326.694381  0.289651 -4.778913  6.092764
     spin moment: {S} , |S| = {    1.80174    0.00437    0.00003 }     1.80174
SCF_NOT_CONV: SCF did not converge  in maximum number of steps.
Geom step, scf iteration, dDmax, dHmax:    0    20      0.289651     0.447810

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      27

siesta: E_KS(eV) =            -2326.6263

siesta: E_KS - E_eggbox =     -2326.6263

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000001    0.272502    0.379898
----------------------------------------
   Max    2.574130
   Res    1.222204    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    2.574130    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):        0.98        0.71      -17.80       -0.21       -0.00       -0.00
(Free)E + p*V (eV/cell)    -2325.2642
Target enthalpy (eV/cell)    -2326.6944

mulliken: Atomic and Orbital Populations:

Species: Fe_nc               

Atom      Orb        Charge      Spin       Svec
----------------------------------------------------------------
    1  1 4s         0.83323   0.14911      0.028   0.005   0.146
    1  2 3dxy       1.57408   0.42224      0.060   0.008   0.418
    1  3 3dyz       1.30280   0.53377     -0.006  -0.000   0.534
    1  4 3dz2       1.36774   0.50393      0.006   0.001   0.504
    1  5 3dxz       1.30286   0.53379     -0.006  -0.000   0.534
    1  6 3dx2-y2    1.57394   0.42240      0.060   0.008   0.418
    1  7 4Ppy       0.01855   0.00268      0.000  -0.000  -0.003
    1  8 4Ppz       0.01328   0.00619      0.005  -0.000  -0.004
    1  9 4Ppx       0.01848   0.00266      0.000  -0.000  -0.003
    1     Total     8.00497   2.54896      0.146   0.020   2.545

    2  1 4s         0.97874   0.04021      0.040  -0.000   0.000
    2  2 3dxy       1.46184   0.50173      0.502   0.000   0.000
    2  3 3dyz       1.27628   0.14217      0.142   0.000  -0.000
    2  4 3dz2       1.25960   0.14517      0.145   0.000  -0.000
    2  5 3dxz       1.27633   0.14217      0.142   0.000  -0.000
    2  6 3dx2-y2    1.46278   0.50096      0.501   0.000   0.000
    2  7 4Ppy       0.08116   0.00937      0.009   0.000   0.000
    2  8 4Ppz       0.11231   0.01813      0.018   0.000   0.000
    2  9 4Ppx       0.08098   0.00935      0.009   0.000   0.000
    2     Total     7.99003   1.50926      1.509   0.002  -0.000

    3  1 4s         0.83324   0.14910      0.028  -0.003  -0.146
    3  2 3dxy       1.57410   0.42222      0.060  -0.007  -0.418
    3  3 3dyz       1.30280   0.53376     -0.006   0.000  -0.534
    3  4 3dz2       1.36774   0.50393      0.006  -0.001  -0.504
    3  5 3dxz       1.30286   0.53379     -0.006   0.000  -0.534
    3  6 3dx2-y2    1.57395   0.42239      0.060  -0.007  -0.418
    3  7 4Ppy       0.01855   0.00268      0.000   0.000   0.003
    3  8 4Ppz       0.01328   0.00619      0.005   0.000   0.004
    3  9 4Ppx       0.01848   0.00266      0.000   0.000   0.003
    3     Total     8.00501   2.54892      0.146  -0.017  -2.545

----------------------------------------------------------------
Total              24.00000   1.80174      1.802   0.004   0.000


Mulliken Atomic Populations:
Atom #   charge [q] valence [e]           S      Sx [e]      Sy [e]      Sz [e]  Species
     1    -0.004968    8.004968    2.548957    0.146074    0.020220    2.544688  Fe_nc
     2     0.009975    7.990025    1.509258    1.509258    0.001561   -0.000003  Fe_nc
     3    -0.005007    8.005007    2.548922    0.146404   -0.017412   -2.544654  Fe_nc
-------------------------------------------------------------------------------
 Total     0.000000                1.801741    1.801736    0.004369    0.000032

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -77.724898
siesta: Eions   =      2102.576618
siesta: Ena     =        92.089899
siesta: Ekin    =      2441.267094
siesta: Enl     =     -1554.181228
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -52.961289
siesta: DUscf   =        10.021970
siesta: DUext   =         0.000000
siesta: Ex      =     -1064.860368
siesta: Ec      =       -95.425734
siesta: Exc     =     -1160.286102
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -2321.897270
siesta: Etot    =     -2326.626274
siesta: FreeEng =     -2326.694381

siesta: Final energy (eV):
siesta:  Band Struct. =     -77.724898
siesta:       Kinetic =    2441.267094
siesta:       Hartree =    1469.391526
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =   -1064.860368
siesta:         Corr. =     -95.425734
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =   -1160.286102
siesta:  Ion-electron =   -4885.311412
siesta:       Ion-ion =    -191.687379
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =   -2326.626274
siesta:         Fermi =      -4.778913

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.000015    0.002929    2.574130
siesta:      2    0.000015    0.264500    0.376074
siesta:      3    0.000001    0.005073   -2.570306
siesta: ----------------------------------------
siesta:    Tot    0.000001    0.272502    0.379898

siesta: Stress tensor (static) (eV/Ang**3):
siesta:     0.000614   -0.000000    0.000000
siesta:    -0.000000    0.000441   -0.000051
siesta:    -0.000000   -0.000130   -0.011109

siesta: Cell volume =        426.771969 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00003650         -0.00004789  Ry/Bohr**3
siesta:           0.00335124         -0.00439694  eV/Ang**3
siesta:           5.36928504         -7.04468176  kBar
(Free)E+ p_basis*V_orbitals  =       -2324.620176
(Free)Eharris+ p_basis*V_orbitals  =       -2319.891173

mulliken: Atomic and Orbital Populations:

Species: Fe_nc               

Atom      Orb        Charge      Spin       Svec
----------------------------------------------------------------
    1  1 4s         0.83323   0.14911      0.028   0.005   0.146
    1  2 3dxy       1.57408   0.42224      0.060   0.008   0.418
    1  3 3dyz       1.30280   0.53377     -0.006  -0.000   0.534
    1  4 3dz2       1.36774   0.50393      0.006   0.001   0.504
    1  5 3dxz       1.30286   0.53379     -0.006  -0.000   0.534
    1  6 3dx2-y2    1.57394   0.42240      0.060   0.008   0.418
    1  7 4Ppy       0.01855   0.00268      0.000  -0.000  -0.003
    1  8 4Ppz       0.01328   0.00619      0.005  -0.000  -0.004
    1  9 4Ppx       0.01848   0.00266      0.000  -0.000  -0.003
    1     Total     8.00497   2.54896      0.146   0.020   2.545

    2  1 4s         0.97874   0.04021      0.040  -0.000   0.000
    2  2 3dxy       1.46184   0.50173      0.502   0.000   0.000
    2  3 3dyz       1.27628   0.14217      0.142   0.000  -0.000
    2  4 3dz2       1.25960   0.14517      0.145   0.000  -0.000
    2  5 3dxz       1.27633   0.14217      0.142   0.000  -0.000
    2  6 3dx2-y2    1.46278   0.50096      0.501   0.000   0.000
    2  7 4Ppy       0.08116   0.00937      0.009   0.000   0.000
    2  8 4Ppz       0.11231   0.01813      0.018   0.000   0.000
    2  9 4Ppx       0.08098   0.00935      0.009   0.000   0.000
    2     Total     7.99003   1.50926      1.509   0.002  -0.000

    3  1 4s         0.83324   0.14910      0.028  -0.003  -0.146
    3  2 3dxy       1.57410   0.42222      0.060  -0.007  -0.418
    3  3 3dyz       1.30280   0.53376     -0.006   0.000  -0.534
    3  4 3dz2       1.36774   0.50393      0.006  -0.001  -0.504
    3  5 3dxz       1.30286   0.53379     -0.006   0.000  -0.534
    3  6 3dx2-y2    1.57395   0.42239      0.060  -0.007  -0.418
    3  7 4Ppy       0.01855   0.00268      0.000   0.000   0.003
    3  8 4Ppz       0.01328   0.00619      0.005   0.000   0.004
    3  9 4Ppx       0.01848   0.00266      0.000   0.000   0.003
    3     Total     8.00501   2.54892      0.146  -0.017  -2.545

----------------------------------------------------------------
Total              24.00000   1.80174      1.802   0.004   0.000


Mulliken Atomic Populations:
Atom #   charge [q] valence [e]           S      Sx [e]      Sy [e]      Sz [e]  Species
     1    -0.004968    8.004968    2.548957    0.146074    0.020220    2.544688  Fe_nc
     2     0.009975    7.990025    1.509258    1.509258    0.001561   -0.000003  Fe_nc
     3    -0.005007    8.005007    2.548922    0.146404   -0.017412   -2.544654  Fe_nc
-------------------------------------------------------------------------------
 Total     0.000000                1.801741    1.801736    0.004369    0.000032

cite: Please see "fe_noncol_sp.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.2.0-15-g8c00451d2
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       2.002
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.995       1.995   100.00
Setup                  1       0.038       0.038     1.91
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.957       1.957    98.07
geom_init              1       0.308       0.308    15.43
state_init             1       0.025       0.025     1.24
hsparse                1       0.000       0.000     0.01
overlap                1       0.023       0.023     1.17
Setup_H0               1       0.206       0.206    10.32
naefs                  2       0.000       0.000     0.01
dnaefs                 2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.011       0.022     1.12
nlefsm                 2       0.065       0.130     6.52
DHSCF_Init             1       0.054       0.054     2.72
DHSCF1                 1       0.006       0.006     0.30
InitMesh               1       0.006       0.006     0.30
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.048       0.048     2.42
REMESH                 1       0.005       0.005     0.23
REORD                196       0.000       0.020     0.99
PHION                  1       0.033       0.033     1.66
COMM_BSC             358       0.000       0.028     1.40
POISON                23       0.002       0.050     2.53
fft                   46       0.001       0.038     1.89
setup_H               21       0.073       1.532    76.81
DHSCF                 22       0.076       1.670    83.68
DHSCF3                22       0.073       1.600    80.17
rhoofdsp              22       0.025       0.548    27.48
XC                    22       0.015       0.321    16.07
GXC-CellXC            22       0.014       0.299    15.00
gridxc@cellXC         22       0.014       0.299    15.00
vmatsp                22       0.028       0.609    30.55
IterSCF               20       0.075       1.505    75.41
compute_dm            20       0.002       0.043     2.17
diagon                20       0.002       0.043     2.16
cdiag                 80       0.000       0.035     1.74
cdiag1                80       0.000       0.004     0.22
cdiag2                80       0.000       0.007     0.33
cdiag3                80       0.000       0.021     1.07
cdiag4                40       0.000       0.002     0.08
MIXER                 20       0.000       0.003     0.14
PostSCF                1       0.144       0.144     7.20
FINAL_HF               1       0.144       0.144     7.20
DHSCF4                 1       0.070       0.070     3.51
dfscf                  1       0.059       0.059     2.97
overfsm                1       0.000       0.000     0.01
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.01
optical                1       0.000       0.000     0.00
  

>> End of run:  17-JAN-2025  12:35:07
Job completed
